applications/salk-portal/src/components/home/Community.jsx

```diff
@@ -164,7 +169,7 @@ const Community = () => {
         <Typography component="h2">Popular experiments</Typography>
         <Grid container item spacing={3}>
           {[1, 2, 3, 4].map((item, i) => (
-            <ExperimentCard i={`communityexperiment_${i}`} heading={"Exploration of the spinal cord"} description={"223 clones"} user={"name"} community={true} />
+            <ExperimentCard experiment={experiment} type="community" />
```

Comment:
for fixed values please use enums or constants instead of strings. this makes the code more maintainable